scripts/start-utils

```diff
@@ -270,14 +228,10 @@ function versionLessThan() {
   oldState=$(shopt -po xtrace || true)
   shopt -u -o xtrace
 
-  # Use if-else since strict mode might be enabled
-  if compare_version "${VERSION}" "lt" "${1?}"; then
-    eval "$oldState"
-    return 0
-  else
-    eval "$oldState"
-    return 1
-  fi
+  eval "$oldState"
+
+  # Verify strict mode because it might be enabled
+  return $(compare_version "${VERSION}" "lt" "${1?}")
```

Comment:
Shouldn't this be the following to avoid returning the output of the `compare_version`?

```suggestion
  compare_version "${VERSION}" "lt" "${1?}"
```